docs/consume-packages/managing-the-global-packages-and-cache-folders.md

```diff
@@ -0,0 +1,116 @@
+ms.reviewer:
+- karann-msft
+- unniravindranathan
+---
+
+# Managing the global packages, cache, and temp folders
+
+Whenever you install, update, or restore a package, NuGet manages packages and package information in several folders outside of your project structure:
+
+| Name | Description and Location (per user)|
+| --- | --- |
+| global&#8209;packages | The *global-packages* folder is where NuGet installs any downloaded package. Each package is fully expanded into a subfolder that matches the package identifier and version number. Projects using the PackageReference format always use packages directly from this folder. When using the `packages.config`, packages are installed to the *global-packages* folder, then copied into the project's `packages` folder.<br/><ul><li>Windows: `%userprofile%\.nuget\packages`</li><li>Mac/Linux: `~/.nuget/packages`</li><li>Override using the NUGET_PACKAGES environment variable, or the `globalPackagesFolder` [configuration setting](../reference/nuget-config-file.md#config-section) (PackageReference only). The environment variable takes precedence over the configuration setting.</li></ul> |
+| http&#8209;cache | The Visual Studio Package Manager (NuGet 3.x+) and the `dotnet` tool store copies of downloaded packages in this cache (saved as `.dat` files), organized into subfolders for each package source. Packages are not expanded, and the cache is not used by the `nuget.exe` tool. The cache has an expiration time of 30 minutes.<br/><ul><li>Windows: `%localappdata%\NuGet\v3-cache`</li><li>Mac/Linux: `~/.local/share/NuGet/v3-cache`</li><li>Override using the NUGET_HTTP_CACHE_PATH environment variable.</li></ul> |
+| temp | A folder where NuGet stores temporary files during its various operations.<br/><li>Windows: `%temp%\NuGetScratch`</li><li>Mac/Linux: `/tmp/NuGetScratch`</li></ul> |
+
+> [!Note]
+> NuGet 3.5 and earlier uses *packages-cache* instead of the *http-cache*, which is located in `%localappdata%\NuGet\Cache`.
+
+By using the cache and *global-packages* folders, NuGet generally avoids downloading packages that already exist on the computer, improving the performance of install, update, and restore operations. When using PackageReference, the *global-packages* folder also avoids keeping downloaded packages inside project folders, where thy might be inadvertently added to source control, and reduces NuGet's overall impact on computer storage.
+
+When asked to retrieve a package, NuGet first looks in the *global-packages* folder. If the exact version of package is not there, then NuGet checks other package sources. For online sources, NuGet checks the cache before retrieving the package over HTTP unless you specify `--no-cache` with `dotnet.exe` commands or `-NoCache` with `nuget.exe` commands.
```

Comment:
it should be something like:
`When asked to retrieve a package, NuGet first looks in the *global-packages* folder. If the exact version of package is not there, then NuGet first checks all the offline package sources. If it can't find the package there, then it will go to all the http sources. For http sources, NuGet checks the cache before retrieving the package over HTTP unless you specify.... `

Reply:
Done